content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/100-cyclic-referential-actions.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Cyclic referential actions with Microsoft SQL Server'
+metaTitle: 'Cyclic referential actions with Microsoft SQL Server'
+metaDescription: 'Circular references can occur between models when each model references another creating a closed loop. Since Microsoft SQL Server does not handle these situations out of the box, learn how to solve this problem.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Circular references can occur between models when each model references another, creating a closed loop. Most SQL databases handle these references out of the box. 
+
+With Microsoft SQL Server, however, if the [referential actions](./) on the relation field is set to something other than `NoAction`, the data model validator will throw an error the cyclic relation.
```

Comment:
... an error about the cyclic relation.

Reply:
I removed the last bit. But i think perhaps it is needed. Will try again